wikidataintegrator/wdi_core.py

```diff
@@ -834,7 +834,7 @@ def get_sitelink(self, site):
         else:
             return None
 
-    def write(self, login, bot_account=True, edit_summary=''):
+    def write(self, login, bot_account=True, edit_summary='', payload_type=u'item'):
```

Comment:
furthermore, payload_type is missleading, as this is a term usually used for http post requests. In the Wikidata context, entity_type would be the right term. Could you adjust that? thx!